docs/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -109,6 +109,16 @@ In `session` mode, this will upload any pending recording data to Sentry. In `bu
 
 Note that it's safe to call `flush()` at any time, even if Session Replay is stopped, in which case, it will do nothing.
 
+## Dead Clicks and Rage Clicks
```

Comment:
I think this might be better fit under the other page also edited, the Replay Issues one

Reply:
Yeah - I was thinking the same - only thing was that when I was actually searching for this information when I was testing an application I first went to look on the Session Replay docs - so I was *trying* to find a good place to put this somewhere that made sense on the Session Replay Product docs page...I agree though in general. I couldn't find a good place for it. So happy to have it not show up here. 

Reasoning:
I notice rage clicks when I'm looking at the Session Replay, so when I have a question about rage clicks my first inclination is to go to the Session Replay docs, where I couldn't find this information or a link to it.

Reply:
I was thinking on the `index.md` of the Session Replay product doc after this section:
https://docs.sentry.io/platforms/javascript/session-replay/#error-linking

Reply:
FWIW the original is from the Troubleshooting on the Session Replay JS SDK docs, so removing it completely from this section would seem strange? 

There are so many entry points into Session Replay docs though, so I don't think I'm the person who would know the right place necessarily for this.

Reply:
If we put this as a new page that is a sibling to `/product/session-replay/performance-overhead/`

Then in the product space we'll have these in the navbar:
```
Issues
∟ Issue Details
    ∟ Rage Click Issues
Session Replay
∟ Dead & Rage Clicks       <- the new page
```
"Rage Click Issues" that we already have, has subheadings: "Detection Criteria, Span Evidence, Fingerprinting" and the new page will have "Dead Clicks and Rage Clicks" only. 


... then we have to cross link the pages which is fine, but it just feels like too many pages.

---

My thought is to put this content directly into `docs/product/issues/issue-details/replay-issues/index.mdx`. 

That little tree structure is probably in the wrong spot to begin with, `Issues > Issue Details -> X` has a lot of stuff that could be better off inside something like "Issues > Issue Types > X`.
Like `Issues > Issue Types > N+1 Queries` or `Issues > Issue Types > Rage Clicks`.
But lets save that for the next PR